### PR TITLE
[WIP] Reference plots

### PR DESCRIPTION
**Pull request by trontrytel**
A py.test way to generate plots from the current state of the model for all test cases. 

The plot_scripts.py file contains all the functions needed to run scampy, read in the data and plot the results. All the remaining test_plot_* files define the simulation requests and plotting tests for py.test.

I'll add the reference simulations next. But before that I would appreciate some feedback on how we want to include the reference simulations into this setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,61 @@
...
 	Zhihong Tan (U. Chicago)
 
-Additional Acknowledgements: 
-	Tapio Schneider (Caltech); 
+Additional Acknowledgements:
+	Tapio Schneider (Caltech);
 	Joao Teixeira (JPL)
-	
+
+# installation #
```
Comment (cmkaul) [inline README.md]: Thanks for adding this information.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,61 @@
...
+```
+
+Generate the turbulence parameters (accepted keywords: defaults, Soares, DYCOMS_RF01, DYCOMS_RF02, Bomex, life_cycle_Tan2018, Rico, TRMM_LBA, ARM_SGP, GATE_III)
+```
+$ python generate_paramlist.py Soares
+```
+
+Compile the source code by running setup.py
+```
+$ CC=mpicc python setup.py build_ext --inplace
```
Comment (cmkaul) [inline README.md]: the CC=mpicc is not actually necessary and/or another C compiler could be specified (in particular no need for mpi)

Comment (trontrytel) [reply README.md]: Right! Sorry, I never even use it with this flag. Must have copied from my earlier notes without thinking. I updated that

**Comment by cmkaul** [comment]
I think this will be great as its own repo!

**Pull request closed without merging**